sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs

```diff
@@ -90,5 +92,34 @@ static TopicRuntimeInfo ParseFromEntryElement(XElement xEntry)
 
             return topicRuntimeInfo;
         }
+
+        public static List<TopicRuntimeInfo> ParseCollectionFromContent(string xml)
+        {
+            try
+            {
+                var xDoc = XElement.Parse(xml);
+                if (!xDoc.IsEmpty)
+                {
+                    if (xDoc.Name.LocalName == "feed")
+                    {
+                        var topicList = new List<TopicRuntimeInfo>();
+
+                        var entryList = xDoc.Elements(XName.Get("entry", ManagementClientConstants.AtomNamespace));
+                        foreach (var entry in entryList)
+                        {
+                            topicList.Add(ParseFromEntryElement(entry));
+                        }
+
+                        return topicList;
+                    }
+                }
+            }
+            catch (Exception ex) when (!(ex is ServiceBusException))
+            {
+                throw new ServiceBusException(false, ex);
+            }
+
+            throw new MessagingEntityNotFoundException("Topic was not found");
```

Comment:
>Topic was not found [](start = 56, length = 19)

nit: No topic was found

Reply:
Same in other classes

Reply:
Fixed

Reply:
Ideally, we would like to have "No topics were found" in collection view, and "topic was not found" when trying to parse a single topic.

Reply:
I've updated the exception messages.